### PR TITLE
Rare Role Rebalancing: Bladesinger, Longbeard, Bulwark, Vaquero (Stats, Skills, Etc)

### PR DESCRIPTION
**Pull request by Optimism333**
## About The Pull Request
Compared to some of the other rare drifter roles at the top of the food chain, Bladesinger, Longbeard, and Vaquero seemed to be falling a bit behind, so I'm putting this pr out as a rebalancing worksheet.

What I've done so far is generally pull their stat increases up to the same level as Lancer/Swordmaster/Bulwark, while also adding some qol in the form of missing skills and description touchups.

Bulwark has been pulled down a touch in heights of strength it can reach. Perhaps it went too far in the name of power fantasy fun.

Open to feedback and suggestions, as always.

### Bladesinger ###
- Bladesinger now gets +2 to strength, +2 to endurance, and +1 to speed, previously they only had +2 to strength and -1 to speed. No bonus to constitution however, leaving them more fragile than the other very rare roles
- Bladesinger gets skilled (3) wrestling skill, previously they had none. All the other rare combat roles have this, it just seemed an oversight
- Bladesinger has had riding and sewing skill removed

### Longbeard ###
- Longbeard gets +2 to strength, +2 to constitution, and +1 to endurance. Previously had no stat bonuses whatsoever
- Longbeards get steelhearted as a trait. Nothing puts fear in the heart of an elder dwarf
- Longbeard description has been touched up slightly: "You've earned your place as one of the old grumblers, a pinnacle of tradition, justice, and willpower. You've come to establish order in these lands, and with your hammer of grudges you'll see it through."
- Longbeards have had riding and sewing skill removed

### Vaquero ###
- Vaquero has been given +1 to strength
- Vaquero has been given dodge expert as a trait, so they can take more advantage of their bonus to speed
- Vaquero has been given expert (4) taming skill, the fact they didn't have this before just seemed a missed opportunity for flavor
- Vaquero has been given skilled (3) athletics, previously had none
- Vaquero has been given skilled (3) wrestling, to mirror other combat classes at this tier
- Vaquero knife and bow skill has been removed
- Vaquero has been given average (2) in whips/flails, and a leather whip
- Vaquero sneaking and stealing have been buffed to skilled (3), and expert (4) respectively
- Vaquero description has been touched up a little, removed reference to Tieberians: "You have been taming beasts of burden all your life, and riding since you were old enough to walk. Perhaps these lands will have use for your skills?"
- Vaquero speed bonus has been reduced to 2 instead of 3
- Reduced Vaquero pick chance to 30, compared to 50 previously

### Sentinel ###
- Sentinel stat bonuses have been reworked. Previously it was: +1 Strength, +1 Constitution, +1 Endurance, +4 Perception, Now it is: +1 Strength, +2 Perception, +1 Speed
- Sentinel has had skilled (3) swords removed
- Sentinel has been given average (2) wrestling and swimming
- Sentinel description has been touched up: "Your overseers have sent you to this distant land as a scout. Your trusted steed, longbow, and spear will allow you to overcome any challenges on the road ahead."
- Reduced Sentinel pick chance to 30, compared to 50 previously

### Immortal Bulwark ###
- Immortal Bulwark has had its +3 to strength reduced to +2, and had its -2 to intelligence removed. It's now standardized with the stats of the other very rare classes
- Immortal Bulwark pick probability is now 15 instead of 10, same as the other very rare classes

## Why It's Good For The Game
- Rebalances some rare classes to make them a little stronger, previously they were falling behind compared to some of the other rare one-slot and race-locked drifter classes at this tier.
- Immortal Bulwark was a touch too strong, it's now standardized with the rest of the very rare classes.
- Sentinel was a bit overtuned, an effective mounted archer, spear user, and also able to use medium armor... while having a much higher pick chance than other rare classes. It's been rebalanced to be a bit more reasonable in stats and skills, as well as rarer overall.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by Optimism333** [comment]
> Vaquero pick rate should be lowered further to something like 20 or 25 since they get expert swords, and riding is a bit faster than it used to be with their high riding skill.
> 
> Also, you should change sentinel pickrate in this PR or I can do it in another one, since they get pretty crazy stats, equipment, and armor training while having a really high pickrate of 50.

I'll look into it when I'm home later. Thanks for the suggestion.

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/vaquero.dm [hunk 1]
```diff
@@ -19,19 +19,21 @@
 
 /datum/outfit/job/roguetown/adventurer/vaquero/pre_equip(mob/living/carbon/human/H)
 	..()
-	H.mind.adjust_skillrank(/datum/skill/combat/knives, 1, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
 	H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/vaquero.dm:24]: A Vaquero with a rapier sounds so unfitting, if it were up to me I'd give them a special whip tbh
Not as broken with a rapier anymore due to the SPD nerf but still quite good

**Comment by ThePainkiller** [comment]
I noticed the sentinel bulwark weapon selection is not weighted, it is entirely random.
I suggest using a weighted pickchance like the one I did for warrior, and assigning values to each loadout, so that the rarity is actually reflected rather than a randomized 67% - 16% - 16% chance.

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/vaquero.dm (hunk 1 above)
Comment (Optimism333) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/vaquero.dm:23]: I gave them a leather whip in addition to the rapier!

**Comment by Optimism333** [comment]
> I noticed the sentinel bulwark weapon selection is not weighted, it is entirely random. I suggest using a weighted pickchance like the one I did for warrior, and assigning values to each loadout, so that the rarity is actually reflected rather than a randomized 67% - 16% - 16% chance.

Made it a weighted list similar to warrior, like you requested.

**Review by ThePainkiller — APPROVED** [review]
More than anything I can't believe they didn't have athletics, huge oversight that I'm glad it's fixed by this PR too

**Pull request merged**